nuxeo-core/nuxeo-core-bulk/src/main/java/org/nuxeo/ecm/core/bulk/BulkService.java

```diff
@@ -26,6 +26,8 @@
 
 /**
  * API to manage Bulk Computation.
+ * <p>
+ * No user check is made by the service, it is the responsability of the caller.
```

Comment:
`responsability` -> `responsibility`

Reply:
"user check" is not clear, does this mean that the command.user is not taken into account when scrolling the db or applying the action ? 
or does it means that command.user is used for some steps but might be submitted by a different non admin user ?